### PR TITLE
Cleaned up code to comply with flake8 linter

### PR DESCRIPTION
**Pull request by taylorjdawson**

### CODE REVIEW

**Comment by AndreMiras** [comment]
Great pull request! Do you mind enforcing the linter check in tox for the fix you did?

File: etherscan/client.py [hunk 1]
```diff
@@ -119,14 +121,16 @@ def connect(self):
...
                     raise EmptyResponse(data.get('message', 'no message'))
-        raise BadRequest("Problem with connection, status code: %s" % req.status_code)
+        raise BadRequest(
+            "Problem with connection, status code: %s" % req.status_code)
 
     def check_and_get_api(self):
         if self.url_dict[self.API_KEY]:  # Check if api_key is empty string
             pass
         else:
-            self.url_dict[self.API_KEY] = input('Please type your EtherScan.io API key: ')
```
Comment (w7788129) [inline etherscan/client.py]: 0xc94770007dda54cF92009BFF0dE90c06F603a09f

File: etherscan/accounts.py [hunk 2]
```diff
@@ -62,7 +67,8 @@ def get_transaction_page(self, page=1, offset=10000, sort='asc', internal=False)
         req = self.connect()
         return req['result']
 
-    def get_all_transactions(self, offset=10000, sort='asc', internal=False) -> list:
+    def get_all_transactions(self, offset=10000, sort='asc',
+                             internal=False) -> list:
         if internal:
             self.url_dict[self.ACTION] = 'txlistinternal'
```
Comment (w7788129) [inline etherscan/accounts.py]: 0xc94770007dda54cF92009BFF0dE90c06F603a09f

File: etherscan/accounts.py [hunk 3]
```diff
@@ -3,6 +3,9 @@
 
 
 class Account(Client):
+    PAGE_NUM_PATTERN = re.compile(
```
Comment (w7788129) [inline etherscan/accounts.py]: 0xc94770007dda54cF92009BFF0dE90c06F603a09f

**Comment by taylorjdawson** [comment]
@AndreMiras You want me to enforce the linter in tox for **all** files not just the test folder? @w7788129 I am not sure what your intentions are but you're cluttering our feed. @corpetty is it possible to remove the unnecessary comments?

**Comment by AndreMiras** [comment]
@taylorjdawson yes exactly. The idea is to enforce tox for at least the fix you made. That way nobody will ever break it again.
So when it's too much work, you can add some exceptions in the `tox.ini` file and as you fix the issue you remove the exception for it. For example I did that recently in https://github.com/kivy/python-for-android/commit/b3d3d45702071fb2fb7d8e44fb5103987d20c4cf#diff-b91f3d5bd63fcd17221b267e851608e8
In this commit you see I fix more stuff and at the same time I remove the error from the `ignore` list.

**Comment by AndreMiras** [comment]
Simply create your new pull request and copy the link here.

**Comment by taylorjdawson** [comment]
Got it. And I just change `commands = flake8 tests/` to `commands = flake8 py-etherscan-api/` correct?

**Comment by AndreMiras** [comment]
It should be `flake8 tests/ etherscan/` and eventually at some point we may also check `examples/` directory. You can always give it a try locally running `tox` command. I'm on the gitter chat today if you want.

**Comment by taylorjdawson** [comment]
PR moved to: #40

**Pull request closed without merging**